riotfile.py

```diff
@@ -1368,12 +1373,12 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="boto",
-            command="pytest {cmdargs} tests/contrib/boto",
+            command="pytest --no-ddtrace {cmdargs} tests/contrib/boto",
             venvs=[Venv(pys=select_pys(max_version="3.6"), pkgs={"boto": latest, "moto": "<1.0.0"})],
         ),
         Venv(
             name="botocore",
-            command="pytest {cmdargs} tests/contrib/botocore",
+            command="pytest --no-ddtrace {cmdargs} tests/contrib/botocore",
```

Comment:
can we document these exceptions (code comments are fine), like, will this never work with `--ddtrace`, or is this something we are just saving as a todo to fix later?

Reply:
Actually I think this one should work, but I got some errors that I wasn't sure if they were failing in other branches, like, but maybe it's actually flaky?:
https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/42049/workflows/7cf2183b-9f81-47eb-ab70-78a7886fa2dd/jobs/2805599/parallel-runs/3?filterBy=FAILED&invite=true#step-108-166
https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/41887/workflows/c1231f57-cb36-4639-b42e-9d599b83d9cc/jobs/2797712?invite=true#step-108-141

Reply:
I believe those were flaky tests that should be fixed by now? Can you try re-running without ITR disabled to verify?